rag_experiment_accelerator/doc_loader/documentLoader.py

```diff
@@ -93,14 +89,12 @@ def load_documents(
         ]
 
         processor = determine_processor(
-            chunking_strategy=chunking_strategy, format=format
+            chunking_strategy=index_config.chunking.chunking_strategy, format=format
         )
```

Comment:
Severity: Major - When passing parameters to `determine_processor`, consider validating the `chunking_strategy` before proceeding, especially since it’s derived from `index_config`. This additional layer would prevent issues caused by invalid configurations that could lead to runtime errors.